synopsys-task/src/synopsys-task/model/coverity.ts

```diff
@@ -20,17 +20,17 @@ export interface AutomationData {
 export interface CoverityConnect {
   connect: CoverityData;
   install?: { directory: string };
-  automation: AutomationData;
-  network: NetworkAirGap;
+  automation?: AutomationData;
+  network?: NetworkAirGap;
   local?: boolean;
   version?: string;
 }
 
 export interface CoverityData {
   user: { name: string; password: string };
   url: string;
-  project: { name: string };
-  stream: { name: string };
+  project: { name?: string };
```

Comment:
Will project.name be optional field or project field itself will be optional and if project field is getting set then name will be mandatory?

If project field is optional, make project field only optional instead of project.name.